packages/neuron-ui/src/components/History/index.tsx

```diff
@@ -30,10 +33,21 @@ const History = () => {
   const [t] = useTranslation()
   const history = useHistory()
   const { search } = useLocation()
+  const [isExporting, setIsExporting] = useState(false)
   const isMainnet = isMainnetUtil(networks, networkID)
 
   const { keywords, onKeywordsChange } = useSearch(search, id, dispatch)
   const onSearch = useCallback(() => history.push(`${RoutePath.History}?keywords=${keywords}`), [history, keywords])
+  const onExport = useCallback(() => {
+    setIsExporting(true)
+    const timer = setTimeout(() => {
```

Comment:
Just curious, why does it need to set a timeout here?

Reply:
The flag `isExporting` is not meant to indicate the completion of exporting the history cuz there's a dialog for that. This flag is used to prevent from double-click on the button.